app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+
+
+def calculate_team_total_rating(players_list: list) -> int:
```

Comment:
list of what?
Also rename players_list to players
no need to add suffix _list
Fix in all places